src/AuthModule/Components/ChangePassword/ChangePassword.tsx

```diff
@@ -1,7 +1,171 @@
-import React from 'react'
+import React, { useContext, useState } from "react";
+import { passwordValidation } from "../Validator/Validator.js";
+import logo from "../../../assets/PMS 3.svg";
+import { Link, useNavigate } from "react-router-dom";
+import axios from "axios";
+import Loader from "../../../SharedMoudule/Components/Loading/Loading";
+import { toast } from "react-toastify";
+import { baseUrl } from "../../../Constants/Components/Urls.js";
+import { useForm } from "react-hook-form";
+import { AuthContext } from "../../../Context/Components/AuthContext.tsx";
 
 export default function ChangePassword() {
+  const [showPassword, setShowPassword] = useState(true);
```

Comment:
Since we have three password inputs, we need three state variables or an object that will hold the state of each input. Currently, if we click in any toggle button, it will flip all three inputs, because they all share the same state.

![toggle password](https://github.com/SalmaAboArab/Project-Management-System/assets/64327685/1748b510-f745-4750-a4f8-18b15d4a6197)